sepa_sale_order_autoconfirm/models/sale_order.py

```diff
@@ -0,0 +1,17 @@
+from odoo import models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    def action_quotation_sent(self):
```

Comment:
this method needs a comment. when is this called and why is it this method that is overridden?

Reply:
done